src/App.js

```diff
@@ -10,15 +10,16 @@ function App() {
         <h1>
           Bookstore CMS
         </h1>
-        <ul className="ul">
+        <ul className="ul p-0 m-0 active-link">
           <li>
-            <Link className="li-app" to="/"> Books</Link>
+            <Link className="li-app" to="/"> BOOKS</Link>
           </li>
           <li>
-            <Link className="li-app" to="/categories">Categories</Link>
+            <Link className="li-app" to="/categories">CATEGORIES</Link>
```

Comment:
- [ ]The color should change when the Books or Categories page is clicked, meaning that if you are on Books page, the book should have a different color, check the design 
![image](https://user-images.githubusercontent.com/62705947/215122734-aeb08c55-7c5e-4b82-ac9e-0fc788816162.png)